features/step_definitions/web_steps.rb

```diff
@@ -57,6 +57,17 @@ def with_scope(locator)
   expect(page).send should, have
 end
 
+Then /^(?:I )should( not)? see the select "([^"]*)"(?: with options "([^"]+)")?$/ do |negate, label, with_options|
+  should = negate ? :not_to        : :to
+  options = (with_options.split(', ') if with_options)
+  expect(page).send should, have_select(label, with_options: options)
```

Comment:
Do we need the negative expectations at all?

Reply:
I can remove them if you want. The nearby steps have them.

Reply:
Yes, let's remove them since these are new steps and we don't need to introduce dead code paths. If there are more dead code paths in other steps, I'm up for removing them as well.

Reply:
done